src/tactic/ui/tools/trigger_wdg.py

```diff
@@ -1791,7 +1791,7 @@ def get_class_name(self):
     def execute(self):
 
         outputs = self.kwargs.get("output")
-        if isinstance(outputs, basestring) and outputs:
+        if isinstance(outputs, str) and outputs:
```

Comment:
In many other places this is done using six:
at the start of the file `import six` than `isinstance(outputs, six.string_types)` is used